skrub/datasets/_utils.py

```diff
@@ -54,3 +81,155 @@ def get_data_dir(name=None, data_home=None):
     if name is not None:
         data_dir = data_dir / name
     return data_dir
+
+
+def load_simple_dataset(dataset_name, data_home=None):
+    """TODO
+    """
+    bunch = _load_dataset_files(dataset_name, data_home)
```

Comment:
total nitpick: maybe we can use variable names a bit more informative than "bunch" (if it was a plain dict we prob. wouldn't call it "dict")